openedx/stanford/common/djangoapps/enrollment/views.py

```diff
@@ -55,38 +55,40 @@ def post(self, request):
         except InvalidKeyError:
             return Response(
                 status=status.HTTP_400_BAD_REQUEST,
-                data={"message": u"Invalid or missing course_id"},
+                data={
+                    'message': u'Invalid or missing course_id',
+                },
             )
-
         if not user_has_role(request.user, CourseStaffRole(course_id)):
             return Response(
                 status=status.HTTP_403_FORBIDDEN,
-                data={"message": u"User does not have permission to update enrollment for [{course_id}].".format(course_id=course_id)},
+                data={
+                    'message': u'User does not have permission to update enrollment for [{course_id}].'.format(
+                        course_id=course_id,
+                    ),
+                },
             )
-
         email = request.data['email']
         try:
             validate_email(email)
         except ValidationError:
             return Response(
                 status=status.HTTP_400_BAD_REQUEST,
-                data={"message": u"Invalid email address"},
+                data={
+                    'message': u'Invalid email address',
+                },
             )
-
         action = request.data['action']
         email_students = request.POST.get('email_students', False) in ['true', 'True', True]
-        auto_enroll = request.POST.get('email_students', False) in ['true', 'True', True]
-
+        auto_enroll = request.POST.get('auto_enroll', False) in ['true', 'True', True]
```

Comment:
Is this the proper fix?

Reply:
oh woops, I must not have saved my local change for this somehow thanks for the catch!